From b1bfbe1790d0e8829aa0c3313ea466e1c1ddbd54 Mon Sep 17 00:00:00 2001
From: David Gibson <dgibson@redhat.com>
Date: Mon, 15 Sep 2014 07:13:33 -0400
Subject: [powerpc] Enable IOMMU and VFIO for ppc64

Message-id: <1410765214-16377-78-git-send-email-dgibson@redhat.com>
Patchwork-id: 94762
O-Subject: [PATCH 77/78] Enable IOMMU and VFIO for ppc64
Bugzilla: 1123145 1123133 1123367
RH-Acked-by: Radim Krcmar <rkrcmar@redhat.com>
RH-Acked-by: Don Zickus <dzickus@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>

This RHEL only patch enables IOMMU and VFIO support in the
configuration for ppc64.

Signed-off-by: David Gibson <dgibson@redhat.com>
---
 create mode 100644 redhat/configs/generic/powerpc64/CONFIG_SPAPR_TCE_IOMMU

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/configs/generic/powerpc64/CONFIG_SPAPR_TCE_IOMMU b/redhat/configs/generic/powerpc64/CONFIG_SPAPR_TCE_IOMMU
new file mode 100644
index 0000000..ffe8303
--- /dev/null
+++ b/redhat/configs/generic/powerpc64/CONFIG_SPAPR_TCE_IOMMU
@@ -0,0 +1 @@
+CONFIG_SPAPR_TCE_IOMMU=y
-- 
1.7.1